### DISCUSSION TITLE
I have no Folder view

### DISCUSSION THREAD

**Discussion by Moosbueffel** [immich-app/immich#13872] [Q&A] [answered]
Trying immich I missed a basically needed feature: a folder view for my external library.

Than I found in the [discussion thread ](https://github.com/immich-app/immich/discussions/5409) and in the [immich documentation](https://immich.app/docs/features/libraries) that there should be a Folder view.

But I can't activate this feature. Is use the lates version 1.119.1. Looking at the documentation it should be found in :
`Users Settings > Features > Folders`

I use the german UI and searched all settings. Users `Settings > Features` not found. `folder activating` not found.  Using the search field for: Datei, Ordner, Directory - nothing! 

Should it be here:
![grafik](https://github.com/user-attachments/assets/fa11d40b-7bef-492a-be99-b12f53d83f82)

Who can help me to find this important feature?

**Comment by bo0tzz** [comment]
https://my.immich.app/user-settings?isOpen=feature+folders

**Reply by Moosbueffel** [reply]
Hmm - the link shows a picture to shared pictures, but the link text let me think, that the feature isn't implemented? Which is contrary to the immich documentation and the thread above!

Or does this mean, I have to do an public link?

**Comment by Moosbueffel** [comment, marked as answer]
I found :). 

In German you found it by clicking on your symbol in the right upper corner. Than:
Kontoeinstellungen - Funktionen - Ordner
